ne16/gvsoc/ne16_gvsoc.h

```diff
@@ -42,13 +42,17 @@ typedef enum ne16_gvsoc_log_level_e {
 static void ne16_gvsoc_log_activate(ne16_dev_t *dev,
                                     ne16_gvsoc_log_level_e log_level,
                                     ne16_gvsoc_log_format_e format) {
+#if __PLATFORM__ == ARCHI_PLATFORM_GVSOC
```

Comment:
This depends on PULP SDK, right? It might be good to add comment in the README for people using this library without PULP SDK.

Reply:
Removed them from here and added them in the tests since the tests are done with pulp-sdk and the library user might not. Commit https://github.com/pulp-platform/pulp-nnx/pull/2/commits/9145445c16ce982fd1215f305389ee0ad68a0fe0